_includes/js/custom.js

```diff
@@ -360,8 +360,12 @@ jQuery(document).ready(function() {
                 if (row.description === undefined) {
                     row.description = ""
                 }
+                if (row.title === undefined) {
```

Comment:
this code could assume that all titles exist since this is a pretty consistent requirement of OBO Foundry

Reply:
Just sticking with the pattern that was already there, but yes, I agree. It should :D